roboteam_ai/src/stp/plays/defensive/DefendPass.cpp

```diff
@@ -137,9 +144,14 @@ void DefendPass::calculateInfoForRobotDefenders() noexcept {
     auto enemyRobots = world->getWorld()->getThem();
     auto enemyClosestToBall = world->getWorld()->getRobotClosestToBall(world::them);
 
+    if (!enemyClosestToBall.has_value()) {
+        roles.at(8) = std::make_unique<role::BallDefender>(role::BallDefender("robot_defender"));
```

Comment:
Is there a possibility to not have the `8` hardcoded?

Reply:
Good one I should be able to change that